cms/src/test/java/uk/nhs/digital/apispecs/CommonmarkMarkdownConverterTest.java

```diff
@@ -241,8 +234,11 @@ private String htmlWithHeadingsAt(final Levels headingsLevels) {
         // <h3 id="heading">Heading</h3>
 
         return headingsLevels.stream()
-            .map(level -> format("<h{0} class=\"{1}\">Heading</h{0}>", level, cssClassForHeadingLevel(level)))
+            .map(level -> level == 2 ? format("<hr class=\"nhsd-a-horizontal-rule\">\n")
```

Comment:
:+1:

Reply:
can we remove the `format` call from this line as it isn't doing anything?

Reply:
Good spot, Harry. Sorted.